### PR TITLE
[R4R] Add S-Lang to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
- This is requirement 1/2 for byobu

### CODE REVIEW

File: recipes/slang/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export CFLAGS="${CFLAGS} ${CPPFLAGS}"
+
+./configure \
+  --prefix=$PREFIX  \
+  --with-readline=gnu \
+  --with-pcre \
+  --with-onig \
+  --with-png \
+  --with-z \
+  --with-iconv=$PREFIX
```
Comment (sodre) [inline recipes/slang/build.sh]: It was picking up the system level one without that line.

**Comment by mariusvniekerk** [comment]
LGTM

File: recipes/slang/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  host:
+    - libiconv
+    - libpng
+    - ncurses
+    - oniguruma 6.9.*
+    - pcre
+    - readline
+    - zlib
+  run:
+    - oniguruma 6.9.*
```
[the remark this answers is not in the document]
Comment (jakirkham) [reply recipes/slang/meta.yaml]: If you'd like to submit a PR to add `run_exports` to `oniguruma`, that would be welcome.

File: recipes/slang/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+test:
+  commands:
+    - slsh --version
+
+about:
+  home: http://www.jedsoft.org/slang
+  license: GPL
```
Comment (jakirkham) [inline recipes/slang/meta.yaml]: Could you please add the version of GPL in the feedstock?

Comment (sodre) [reply recipes/slang/meta.yaml]: No problem, that is taken care in conda-forge/slang-feedstock#2.

File: recipes/slang/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (sodre) [reply recipes/slang/meta.yaml]: Please see conda-forge/origuruma-feedstock#25...

**Pull request merged**